nilearn/datasets/atlas.py

```diff
@@ -863,3 +863,77 @@ def fetch_atlas_allen_2011(data_dir=None, url=None, resume=True, verbose=1):
     params.extend(list(zip(keys, sub_files)))
 
     return Bunch(**dict(params))
+
+
+def fetch_atlas_surf_destrieux(data_dir=None, url=None,
+                               resume=True, verbose=1):
+    """Download and load Destrieux et al, 2010 cortical atlas.
+
+    This atlas returns 74 sulco-gyral structures per hemisphere distributed
+    with Freesurfer in fsaverage5 surface space.
+
+    Parameters
+    ----------
+    data_dir: str, optional
+        Path of the data directory. Use to force data storage in a non-
+        standard location. Default: None
+
+    url: str, optional
+        Download URL of the dataset. Overwrite the default URL.
+    resume: bool, optional (default True)
+        If True, try resuming download if possible.
+
+    verbose: int, optional (default 1)
+        Defines the level of verbosity of the output.
+
+    Returns
+    -------
+        data: sklearn.datasets.base.Bunch
+        dictionary-like object, contains:
+        - 'labels' : list
```

Comment:
I just saw other fetchers. They have return arguments in double quotes but not single quotes. May be that's why sphix fails. Can you also correct them ?